.github/workflows/continuous-integration-workflow.yml

```diff
@@ -63,8 +63,10 @@ jobs:
             go-mod-${{ env.CACHE_VERSION }}
       - name: Download Go vendor packages
         run: go mod download
-      - name: Set up Yarn for Contracts
-        run: yarn setup:contracts
+      - name: Set up Yarn for Contracts and compile
+        run: yarn setup:contracts && ./evm-contracts/scripts/native_solc_compile_all
```

Comment:
I don't think we need `yarn setup:contracts &&` here anymore since the belt artifacts aren't used in these tests now.

`./evm-contracts/scripts/native_solc_compile_all` should suffice (and reduce CI time)